package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "webmaker.org",
   "version": "1.1.129",
+  "engines": {
```

Comment:
Reminder for myself to remove this chunk before merging this PR.  This is for the staging on Heroku to work.